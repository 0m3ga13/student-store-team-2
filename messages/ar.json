{
    "Index": {
        "Title": "المتجر الصغير",
        "Home": "الرئيسية",
        "Products": "المنتجات",
        "Orders": "الطلبيات",
        "State": "الحالة",
        "Buy": "شراء",
        "Borrow": "إعارة",
        "Description": "هذه المنصة هي مربحة للجميع: القرضاء يحققون دخلاً إضافيًا، والمقترضون يقللون التكاليف.",
        "Profile": "الحسـاب",
        "For Sale": "للبيع",
        "For Borrow": "للإعارة",
        "Sale & Borrow": "إعارة وشراء",
        "Day": "لليوم",
        "How It Works": "طريـقة العمل",
        "Hello": "مرحبا",
        "Sign out": "تسجيل الخروج",
        "Sign In": "تسجيل الدخول",
        "Login with Google": "سحل الدخول بحسابك",
        "Sign Up With Google": "سحل الدخول بحسابك"
    }
}
