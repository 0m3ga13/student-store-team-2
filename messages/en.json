{
    "Index": {
        "Title": "MiniStore",
        "Home": "Home",
        "Products": "Products",
        "Orders": "Orders",
        "State": "State",
        "Donate": "Donate",
        "Buy": "Buy",
        "Borrow": "Borrow",
        "Description": "This platform is a win-win for everyone involved lenders earn extra income, borrowers cut costs.",
        "Profile": "Profile",
        "for_sale": "for_sale",
        "for_borrow": "for_borrow",
        "Sale & Borrow": "Sale & Borrow",
        "Day": "Day",
        "How It Works": "How It Works",
        "recently added": "recently added",
        "Hello": "Hello",
        "Sign out": "Sign out",
        "Sign In": "Sign In",
        "Login with Google": "Login with Google",
        "Sign Up With Google": "Sign Up With Google",
        "Free delivery": "Free delivery",
        "feature descriprion 1": "You can receive your order without incurring any additional shipping costs",
        "qualite guarantee": "qualite guarantee",
        "feature descriprion 2": "Our products or services meet high standards and perform as promised.",
        "daily offers": "daily offers",
        "feature descriprion 3": "Exclusive discounts and promotions on a daily basis",
        "100% secure payement": "100% secure payement",
        "feature descriprion 4": "Your financial information is protected and your transactions are safe from unauthorized access",
        "My Dashboard": "My Dashboard",
        "Awaiting Pickup": "Awaiting Pickup",
        "All": "All",
        "Active": "Active",
        "Completed": "Completed",
        "Sort": "Sort:",
        "Newest": "Newest",
        "Oldest": "Oldest",
        "my stuff":"my stuff",
        "borrowings": "borrowings",
        "lendings": "lendings",
        "purchases": "purchases",
        "sales": "sales",
        "My Borrowings": "My Borrowings",
        "My Lendings": "My Lendings",
        "My Purchases": "My Purchases",
        "My Sales": "My Sales",
        "General details": "General details",
        "First Name": "First Name",
        "Last Name": "Last Name",
        "Birth Date": "Birth Date",
        "Gender": "Gender",
        "Institution": "Institution",
        "Contact & address Detail": "Contact & address Detail",
        "Phone Number": "Phone Number",
        "Building": "Building",
        "Street": "Street",
        "City": "City",
        "State_place": "State",
        "Save Changes": "Save Changes",
        "Upload Image": "Upload Image",
        "Close": "Close",
        "Upload": "Upload",
        "Loading": "Loading",
        "Not a member ?": "Not a member ?",
        "Sign Up": "Sign Up",
        "Store": "Store",
        "Create Your": "Create Your",
        "and start exchanging with other students": "and start exchanging with other students",
        "Or Login Using:": "Or Login Using:",
        "Login": "Login",
        "LOGIN": "LOGIN ",
        "Already a member ?": "Already a member ?",
        "Email and password are required": "Email and password are required",
        "Password should be at least 8 characters long": "Password should be at least 8 characters long",
        "and Conditions and Privacy Policy of The website": "and Conditions and Privacy Policy of The website",
        "Email": "Email",
        "Password": "Password",
        "Full Name": "Full Name",
        "Address": "Address",
        "and Privacy Policy": "and Privacy Policy",
        "I have read and agree with all of the Terms": "I have read and agree with all of the Terms",
        "PROFILE": "PROFILE",
        "SECURITY": "SECURITY",
        "Upload ID": "Upload ID",
        "Edit ID": "Edit ID",
        "Edit Image": "Edit Image",
        "WHY DONATE": "WHY DONATE",
        "MAKE A DIFFERENCE BY DONATING": "MAKE A DIFFERENCE BY DONATING",
        "Donate paragraph": "In our fast-paced lives, it's easy to forget the power of gratitude. Students, often on tight budgets, rely on our support. Your donation is more than just financial aid; it's a symbol of gratitude. By giving generously, you're helping students create a brighter future, one contribution at a time.",
        "Small Help": "Small Help",
        "Some Help": "Some Help",
        "Big Help": "Big Help",
        "List an Item": "List an Item",
        "Drag and drop some files here or click to select files": "Drag and drop some files here or click to select files",
        "Location": "Location",
        "Category": "Category",
        "books": "books",
        "kitchen": "kitchen",
        "sports": "sports",
        "games": "games",
        "clothes": "clothes",
        "electronics": "electronics",
        "furniture": "furniture",
        "description": "Description",
        "Product Condition": "Product Condition",
        "Sell": "Sell",
        "Price Per Week": "Price Per Week",
        "Price": "Price",
        "Upload Image(s)": "Upload Image(s)",
        "List": "List",
        "Product Name": "Product Name",
        "Comments": "Comments",
        "Type your comment": "Type your comment",
        "Type your reply": "Type your reply",
        "Respond": "Respond",
        "Like": "Like",
        "Send": "Send",
        "Authentication error": "Invalid login credentials. Please check your email and password.",
        "Filter": "Filter",
        "Pending": "Pending",
        "Quick Links": "Quick Links",
        "About LACITÉ": "ABOUT LACITÉ",
        "About Us": "ABOUT US",
        "Github Project Link": "GITHUB PROJECT LINK",
        "Contact Us": "Contact Us",
        "We Couldn't Find the page you're looking for": "We Couldn't Find the page you're looking for",
        "Go To Home": "Go To Home",
        "We Couldn't Find the requested Product": "We Couldn't Find the requested Product."
    }
}
