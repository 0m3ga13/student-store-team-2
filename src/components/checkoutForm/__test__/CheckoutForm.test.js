import { NextIntlClientProvider } from "next-intl";
import renderer from "react-test-renderer";

import CheckoutForm from "../CheckoutForm";
import messages from "../../../../messages/en.json";

it("renders correctly", () => {
    const tree = renderer
        .create(
            <NextIntlClientProvider locale='en' messages={messages}>
                <CheckoutForm />
            </NextIntlClientProvider>
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});
