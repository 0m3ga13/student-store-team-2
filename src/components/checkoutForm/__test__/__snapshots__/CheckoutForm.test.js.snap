// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="px-3 md:w-8/12 w-4/5"
>
  <div
    className="w-full mx-auto rounded-lg bg-white border border-gray-200 p-3 text-gray-800 font-light mb-6"
  >
    <div
      className="w-full flex mb-3 items-center"
    >
      <div
        className="w-32"
      >
        <span
          className="text-gray-600 font-semibold"
        >
          Contact
        </span>
      </div>
      <div
        className="flex-grow pl-3"
      >
        <span>
          Scott Windon
        </span>
      </div>
    </div>
    <div
      className="w-full flex items-center"
    >
      <div
        className="w-32"
      >
        <span
          className="text-gray-600 font-semibold"
        >
          Billing Address
        </span>
      </div>
      <div
        className="flex-grow pl-3"
      >
        <span>
          123 George Street, Sydney, NSW 2000 Australia
        </span>
      </div>
    </div>
  </div>
  <div
    className="w-full mx-auto rounded-lg bg-white border border-gray-200 text-gray-800 font-light mb-6"
  >
    <div
      className="w-full p-3 border-b border-gray-200"
    >
      <div
        className="mb-5"
      >
        <label
          className="flex items-center cursor-pointer"
          htmlFor="type1"
        >
          <input
            className="form-radio h-5 w-5 text-indigo-500"
            id="type1"
            name="type"
            type="radio"
          />
          <img
            alt="Type 1"
            className="h-6 ml-3"
            data-nimg="1"
            decoding="async"
            height={40}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/_next/image?url=https%3A%2F%2Fleadershipmemphis.org%2Fwp-content%2Fuploads%2F2020%2F08%2F780370.png&w=384&q=75"
            srcSet="/_next/image?url=https%3A%2F%2Fleadershipmemphis.org%2Fwp-content%2Fuploads%2F2020%2F08%2F780370.png&w=256&q=75 1x, /_next/image?url=https%3A%2F%2Fleadershipmemphis.org%2Fwp-content%2Fuploads%2F2020%2F08%2F780370.png&w=384&q=75 2x"
            style={
              Object {
                "color": "transparent",
              }
            }
            width={160}
          />
        </label>
      </div>
      <div>
        <div
          className="mb-3"
        >
          <label
            className="text-gray-600 font-semibold text-sm mb-2 ml-1"
          >
            Name on card
          </label>
          <div>
            <input
              className="w-full px-3 py-2 mb-1 border border-gray-200 rounded-md focus:outline-none focus:border-indigo-500 transition-colors"
              placeholder="John Smith"
              type="text"
            />
          </div>
        </div>
        <div
          className="mb-3"
        >
          <label
            className="text-gray-600 font-semibold text-sm mb-2 ml-1"
          >
            Card number
          </label>
          <div>
            <input
              className="w-full px-3 py-2 mb-1 border border-gray-200 rounded-md focus:outline-none focus:border-indigo-500 transition-colors"
              placeholder="0000 0000 0000 0000"
              type="text"
            />
          </div>
        </div>
        <div
          className="mb-3 -mx-2 flex items-end"
        >
          <div
            className="px-2 w-1/4"
          >
            <label
              className="text-gray-600 font-semibold text-sm mb-2 ml-1"
            >
              Expiration date
            </label>
            <div>
              <select
                className="form-select w-full px-3 py-2 mb-1 border border-gray-200 rounded-md focus:outline-none focus:border-indigo-500 transition-colors cursor-pointer"
              >
                <option
                  value="01"
                >
                  01 - January
                </option>
                <option
                  value="02"
                >
                  02 - February
                </option>
                <option
                  value="03"
                >
                  03 - March
                </option>
                <option
                  value="04"
                >
                  04 - April
                </option>
                <option
                  value="05"
                >
                  05 - May
                </option>
                <option
                  value="06"
                >
                  06 - June
                </option>
                <option
                  value="07"
                >
                  07 - July
                </option>
                <option
                  value="08"
                >
                  08 - August
                </option>
                <option
                  value="09"
                >
                  09 - September
                </option>
                <option
                  value="10"
                >
                  10 - October
                </option>
                <option
                  value="11"
                >
                  11 - November
                </option>
                <option
                  value="12"
                >
                  12 - December
                </option>
              </select>
            </div>
          </div>
          <div
            className="px-2 w-1/4"
          >
            <select
              className="form-select w-full px-3 py-2 mb-1 border border-gray-200 rounded-md focus:outline-none focus:border-indigo-500 transition-colors cursor-pointer"
            >
              <option
                value="2020"
              >
                2020
              </option>
              <option
                value="2021"
              >
                2021
              </option>
              <option
                value="2022"
              >
                2022
              </option>
              <option
                value="2023"
              >
                2023
              </option>
              <option
                value="2024"
              >
                2024
              </option>
              <option
                value="2025"
              >
                2025
              </option>
              <option
                value="2026"
              >
                2026
              </option>
              <option
                value="2027"
              >
                2027
              </option>
              <option
                value="2028"
              >
                2028
              </option>
              <option
                value="2029"
              >
                2029
              </option>
            </select>
          </div>
          <div
            className="px-2 w-1/4"
          >
            <label
              className="text-gray-600 font-semibold text-sm mb-2 ml-1"
            >
              Security code
            </label>
            <div>
              <input
                className="w-full px-3 py-2 mb-1 border border-gray-200 rounded-md focus:outline-none focus:border-indigo-500 transition-colors"
                placeholder="000"
                type="text"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="w-full p-3"
    >
      <label
        className="flex items-center cursor-pointer"
        htmlFor="type2"
      >
        <input
          className="form-radio h-5 w-5 text-indigo-500"
          id="type2"
          name="type"
          type="radio"
        />
        <img
          alt="Type 2"
          className="ml-3"
          data-nimg="1"
          decoding="async"
          height={40}
          loading="lazy"
          onError={[Function]}
          onLoad={[Function]}
          src="https://upload.wikimedia.org/wikipedia/commons/b/b5/PayPal.svg"
          style={
            Object {
              "color": "transparent",
            }
          }
          width={80}
        />
      </label>
    </div>
  </div>
  <div>
    <button
      className="block w-full max-w-xs mx-auto bg-accent hover:bg-cyan-800 focus:bg-cyan-950 text-white rounded-lg px-3 py-2 font-semibold"
    >
      <i
        className="mdi mdi-lock-outline mr-1"
      />
       DONATE AMOUNT
    </button>
  </div>
</div>
`;
